site/_posts/2019-04-02-0.13.0-release.md

```diff
@@ -0,0 +1,169 @@
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 0.13.0 release. This is the
+covers more than 2 months of development work and includes [**550 resolved
+issues**][1] from [**81 distinct contributors**][22].
+
+See the [Install Page][2] to learn how to get the libraries for your
+platform. The [complete changelog][3] is also available.
+
+While it's a large release, this post will give some brief highlights in the
+project since the 0.12.0 release from January.
+
+## New committer and PMC member
+
+The Arrow team is growing! Since the 0.12.0 release we have increased the size
+of our committer and PMC rosters.
+
+* [Andy Grove][13] was promoted to PMC member
+* [Micah Kornfield][14] was added as a committer
+
+Thank you for all your contributions!
+
+## Rust DataFusion Query Engine donation
+
+Since the last release, we received a donation of [DataFusion][17], a
+Rust-native query engine for the Arrow columnar format, whose development had
+been led prior by Andy grove. [Read more about DataFusion][19] in our February
+blog post.
+
+This is an exciting development for the Rust community, and we look forward to
+developing more analytical query processing within the Apache Arrow project.
+
+## Arrow Flight gRPC progress
+
+Over the last couple months, we have made significant progress on Arrow Flight,
+an Arrow-native data messaging framework. We have have integration tests to
+check C++ and Java compatibility, an we have added Python bindings for the C++
+library. We will write a future blog post to go into more detail about how
+Flight works.
+
+## C++ notes
+
+There were 231 issues relating to C++ in this release, far too much to
+summarize in a blog post. Some notable items include:
+
+* An experimental `ExtensionType` was developed for creating user-defined data
+  types that can be embedded in the Arrow binary protocol. This is not yet
+  finalized, but [feedback would be welcome][51].
+* We have undertaken a significant reworking of our CMake build system for C++
+  to make the third party dependencies more configurable. Among other things,
+  this eases work on packaging for Linux distributions. Read more about this in
+  the [C++ developer documentation][18].
+* Laying more groundwork for an Arrow-native in-memory query engine
+* We began building a reader for line-delimited JSON files
+* Gandiva can now be compiled on Windows with Visual Studio
+
+## C# Notes
+
+C# .NET development has picked up since the initial code donation last
+fall. 11 issues were resolved this release cycle.
+
+The Arrow C# package is now available via NuGet.
```

Comment:
Here is the link to the package: https://www.nuget.org/packages/Apache.Arrow/0.13.0